src/harness/testcases/spectrum_inquiry_testcase.py

```diff
@@ -30,6 +30,105 @@ def tearDown(self):
     pass
 
   @winnforum_testcase
+  def test_WINNF_FT_S_SIQ_4(self):
+    """Send Spectrum Inquiry from inside coastal exclusion zone.
+       With no ESC deployed, use a Category A device to request Spectrum
+       Inquiry.
+
+    The response should be Success, with NO channels in result.
+    """
+    # Register the device
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    self._sas_admin.InjectFccId({'fccId': device_a['fccId']})
+    # Locate this device close to coast (Half Moon Bay, Calif.)
+    device_a['installationParam']['latitude'] = 37.444267
+    device_a['installationParam']['longitude'] = -122.428628
+    request = {'registrationRequest': [device_a]}
+    response = self._sas.Registration(request)['registrationResponse'][0]
+    cbsd_id = response['cbsdId']
+    del request, response
+
+    # Query for the spectrum affected by the coastal exclusion zone, i.e.
+    # 3550-3650 MHz.
+    spectrum_inquiry_0 = {
+        'cbsdId': cbsd_id,
+        'inquiredSpectrum': [{
+            'lowFrequency': 3550000000.0,
+            'highFrequency': 3650000000.0
+        }]
+    }
+    request = {'spectrumInquiryRequest': [spectrum_inquiry_0]}
+    # Check spectrum inquiry response
+    response = self._sas.SpectrumInquiry(request)['spectrumInquiryResponse'][0]
+    self.assertEqual(response['cbsdId'], cbsd_id)
+    self.assertEqual(len(response['availableChannel']), 0)
+    self.assertEqual(response['response']['responseCode'], 0)
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_SIQ_5(self):
+    """Send Spectrum Inquiry for coastal exclusion zone spectrum.
+
+    Category B device is located inside, then outside of the coastal exclusion
+    zone and inquires for spectrum reserved for coastal exclusion zone.
+
+    The response should be Success, with NO channels in result.
+    """
+    # Register the device
+    device_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    self.assertEqual(device_b['cbsdCategory'], 'B')
+    self._sas_admin.InjectFccId({'fccId': device_b['fccId']})
+    # Locate this device close to coast (Half Moon Bay, Calif.)
+    device_b['installationParam']['latitude'] = 37.444267
+    device_b['installationParam']['longitude'] = -122.428628
+    request = {'registrationRequest': [device_b]}
+    response = self._sas.Registration(request)['registrationResponse'][0]
```

Comment:
Check for Registration ResponseCode
   ` self.assertEqual(response['response']['responseCode'], 0)
`